src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -610,8 +610,8 @@ static int uct_cuda_copy_detect_vmm(void *address,
         *vmm_mem_type = UCS_MEMORY_TYPE_CUDA;
     }
 
-err:
-    UCT_CUDADRV_FUNC_LOG_DEBUG(cuMemRelease(alloc_handle));
+out:
+    UCT_CUDADRV_FUNC_LOG_WARN(cuMemRelease(alloc_handle));
```

Comment:
does it mean we also expect `cuMemGetAllocationPropertiesFromHandle` to be always sucessfull? Because it uses `UCT_CUDADRV_FUNC_LOG_DEBUG`